python/ray/monitor.py

```diff
@@ -357,6 +360,7 @@ def run(self):
         try:
             self._run()
         except Exception:
+            logger.exception("Error in monitor loop")
```

Comment:
If you don't do this, the error message is lost forever trying to terminate nodes.

Reply:
Does this mean nodes probably weren't cleaned up? If so, would be good to print that in the error message.